sbncode/Calibration/fcl/sbnd/sbnd_trackcalo_skimmer.fcl

```diff
@@ -0,0 +1,106 @@
+BEGIN_PROLOG
+
+caloskim_pandoraCalo: {
+  CaloAlg: {
+    CalAmpConstants: [ 1., 1., 1.]
+    CalAreaConstants: [ 1., 1., 1.]
+    CaloDoLifeTimeCorrection: false
+    CaloLifeTimeForm: 0
+    CaloUseModBox: true
+  }
+  CorrectSCE: false
+  FlipTrack_dQdx: false
+  SpacePointModuleLabel: "pandora"
+  T0ModuleLabel: ""
+  TrackModuleLabel: "pandoraTrack"
+  UseArea: true
+  module_type: "Calorimetry"
```

Comment:
Maybe more of a question for @gputnam than anything else, but is there a raeson not to use the `Gnocci` calorimetry module in both cases? It seems to me like it would be less confusiong to use that in all cases rather than use different modules for the sce corrected and uncorrected

Reply:
@etyley I wrote the fcl for sbnd. I put two calo modules just to be safe since I checked the sbnd calorimetry and saw two modules for sce uncorrected and corrected. I can edit the fcl to use the Gnocchi one to keep it consistent.

Reply:
yeah probably best to use Gnocchi